addons/photo_collage/views/photo_collage.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+        <record model="ir.ui.view" id="photo_collage_form">
+            <field name="name">photo_collage.form</field>
+            <field name="model">photo_collage</field>
+            <field name="arch" type="xml">
+                <form>
+                    <sheet>
+                        <div class="oe_title">
+                            <label for="name" class="oe_edit_only"/>
+                            <h1 class="oe_inline">
+                                <field name="name"/>
+                            </h1>
+                        </div>
+                        <group col="2">
+                            <label for="attachment"/>
+                            <div>
+                                <field name="attachment" filename="filename" class="no_clear_button"/>
+                                <div class="oe_grey oe_text_right oe_edit_only">Max. wielkość pliku do <strong>25 MB</strong></div>
```

Comment:
I took it from bestja_files, there is such a limit. If you would like to have it changed, you may need to change it there as well.